practice-app/routes/exchange_rate_api.js

```diff
@@ -65,4 +81,47 @@ function sendRequestToEndpoint(res, from, to) {
     });
 }
 
-module.exports = router;
+function calculateAverage(res, start_date, end_date, from, to) {
+    const reqOptions = {'url': process.env.EXCHANGE_RATE_API_HISTORY_URL, 'qs': {'start_at': start_date, 'end_at':end_date, 'base': from, 'symbols': to}};
+    request(reqOptions, (err, response, body) => {
+        if (err) {
+            // If something went wrong during the request, send it to the user
+            res.status(400).send({'error': '' + err});
+        } else if (response.statusCode !== 200) {
+            // If the endpoint returns an error code (like invalid from/to symbols)
+            // send it directly to the user
+            res.status(400).send(body);
```

Comment:
`send(body)` should be `send(JSON.parse(body))`. The former one sends text in case of error. We want to send JSON objects everytime